src/dune_pkg/solver_env.ml

```diff
@@ -210,49 +210,58 @@ type t =
   { flags : Variable.Flag.Set.t
   ; sys : Variable.Sys.Bindings.t
   ; const : Variable.Const.Bindings.t
+  ; repos : Workspace.Repository.Name.t list
   }
 
 module Fields = struct
   let flags = "flags"
   let sys = "sys"
   let const = "const"
+  let repos = "repositories"
 end
 
 let default =
   { flags = Variable.Flag.Set.all
   ; sys = Variable.Sys.Bindings.empty
   ; const = Variable.Const.bindings
+  ; repos = [ Workspace.Repository.Name.of_string ":standard" ] (* TODO *)
```

Comment:
What needs to be done here?

Reply:
I believe there is a nicer way to implement `:standard` that aligns with the rest of Dune, but here it's kinda cheated in by using `:standard` as a name of the repo and including it in the repo list implicitly.

Reply:
Yes, it should be using `Ordered_set_language` here.